cmd/meroxa/root/billing/billing.go

```diff
@@ -14,32 +14,39 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package root
+package billing
 
 import (
-	"fmt"
+	"context"
 
-	"github.com/meroxa/cli/cmd/meroxa/root/deprecated"
+	"github.com/meroxa/cli/cmd/meroxa/builder"
 
-	"github.com/spf13/cobra"
+	"github.com/meroxa/cli/cmd/meroxa/root/open"
 )
 
-// TODO: Check how to disable parent flags (e.g.: --json)
+type Billing struct{}
```

Comment:
Why do we need a separate command for this if it's doing the same thing? We could just build the same command at a different place (under root instead of under open).

If we really want a separate command we could just do:
```
type Billing struct {
	open.Billing
}
```

And override any methods if needed (doesn't seem like we need to override anything).

Reply:
@lovromazgon good point.